### ISSUE TITLE
caret nnet model warning

### ISSUE THREAD

**Issue by amylee-lixinyi** [topepo/caret#905]
I‘m trying to construct a neural network regression model for my data set and I got this warning message as following:
_Warning message:
In nominalTrainWorkflow(x = x, y = y, wts = weights, info = trainInfo,  :
  There were missing values in resampled performance measures._
I did googled for related questions but didn't find a clear answer about how to solve it or whether it's highly impact to the regression results. 

I did the following checking already,
1) I checked my NNET model for prediction, it do not predict a constant value( this seems to happened to a lot of other user who faced similar warning message) .
2) I set linout = TRUE for the nnet model to tell it's a numerical regression problem
3) the predictors had been center and scaled use preProc = c("center", "scale")

I saw this  #issue 801 (https://github.com/topepo/caret/issues/801) and find out the same warning pop out in the final answer sector, which make me feel even more confused.

my training code looks like this,

library(caret)
fit_control <- trainControl(method = "repeatedcv", number = 10,repeats=20)
set.seed(12345)
ANN_fit<- train(v1 ~ ., data = mydata,method = "nnet",trControl = fit_control, tuneLength = 5, preProc = c("center", "scale"),linout = TRUE)
ANN_fit

My data include 6 independent numeric predictors and one numeric dependent variable, more than 700 set of data in total.

If you have any experience about related issue, your advice and help are really appreciated. Thanks a lot!

**Comment by amylee-lixinyi** [comment]
Thanks a lot for your quick reply.

I checked my data and find no NA existed, so I don't think this is the main
reason for the warning message?

The warning message is

*Warning message:*
*In nominalTrainWorkflow(x = x, y = y, wts = weights, info = trainInfo,  :*
*  There were missing values in resampled performance measures.*

Attached both the R code and the data set to reproduce the warning.

Max Kuhn <notifications@github.com> 于2018年6月25日周一 下午9:25写道：

**Comment by topepo** [comment]
They weren't attached.

**Comment by amylee-lixinyi** [comment]
Thanks for your response.

Sorry, I was reply through Email, maybe that's why the attachments gone missing.
RE-attach the files via web page. 


[ANN warning.zip](https://github.com/topepo/caret/files/2151530/ANN.warning.zip)

**Comment by topepo** [comment]
It looks like it happens when you have one hidden unit and almost no regularization. What is happening is that the model is predicting a value very close to a constant (so that the RMSE is a little worse than the basic st deviation of the outcome):

```r
> ANN_cooling_fit$resample %>% dplyr::filter(is.na(Rsquared))
      RMSE Rsquared      MAE size decay     Resample
1 8.414010       NA 6.704311    1 0e+00 Fold04.Rep01
2 8.421244       NA 6.844363    1 0e+00 Fold01.Rep03
3 7.855925       NA 6.372947    1 1e-04 Fold10.Rep07
4 7.963816       NA 6.428947    1 0e+00 Fold07.Rep09
5 8.492898       NA 6.901842    1 0e+00 Fold09.Rep09
6 7.892527       NA 6.479474    1 0e+00 Fold10.Rep10
> sd(mydata$V7)
[1] 7.962888
```

So it's nothing to really worry about; just some parameters that do very poorly.

**Comment by amylee-lixinyi** [comment]
Thanks for your quick response!

**Comment by datadocSharma** [comment]
Is there a way to see resampling results (for those samples for which computations could be made) when the following error message occurs?
_Warning message:
In nominalTrainWorkflow(x = x, y = y, wts = weights, info = trainInfo, :
There were missing values in resampled performance measures.__

Say I specified 5 folds, and 4 of them worked while the 5th failed. Can I check the results of 4? I am using parallel processing on a linux AMI, so I do not even get to see normal +Fold.O, -Fold.1...etc. 

TIA!
